cmake/modules/hunter_autotools_project.cmake

```diff
@@ -98,7 +98,7 @@ function(hunter_autotools_project target_name)
 
   if(PARAM_UNPARSED_ARGUMENTS)
     hunter_internal_error(
-        "Invalid arguments passed to hunter_autotools_configure:"
+        "Invalid arguments passed to hunter_autotools_project:"
```

Comment:
What do you think of https://github.com/ruslo/hunter/issues/837 ? I can bump `odb-boost`, but me feeling is that it wastes disk space on the cache server since the packages shouldn't change. Anyway, I'll do as you see best fit